spec/app_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'spec_helper'
+require 'rack/test'
+require 'omniauth-myusa'
+require 'stub_classes'
+
+describe 'ComplianceViewer' do
+  include Rack::Test::Methods
+
+  class StubComplianceData
+  end
+
+  def app
+    ComplianceViewer.new! StubComplianceData.new
```

Comment:
Hoo boy. Mind adding that as an inline comment? If the tests weren't passing, I would have assumed it was a typo.